.github/workflows/release.yaml

```diff
@@ -109,7 +109,11 @@ jobs:
             ${{ runner.os }}-cargo-release-
 
       - name: Build package
-        run: cargo build --release --target ${{ matrix.target }}
+        run: cargo build --features=snowflake --release --target ${{ matrix.target }}
```

Comment:
Do we want snowflake?

Reply:
I reverted the changes prior to my merge, we can remove if not needed

Reply:
removed